### DISCUSSION TITLE
Datagrid current page items list

### DISCUSSION THREAD

**Discussion by dockinkong** [Megabit/Blazorise#3280] [Q&A] [answered]
Hello, 
Is there a way to get current page items in a list please?
I would like to fire some requests to get complementary information but only on visible items...

Regards and HAPPY NEW YEAR !

**Comment by stsrki** [comment, marked as answer]
We have exposed this API recently on DataGrid #3244. But, it is going to be possible in v1.0 only.

PS. All the best to you in the new year :)

**Reply by dockinkong** [reply]
Thanks a lot
